ttnn/cpp/ttnn/operations/normalization/softmax/device/kernels/compute/softmax.cpp

```diff
@@ -62,21 +67,53 @@ void MAIN {
     uint32_t ht = start_ht;
     bool wait_mask = true;
     for (uint32_t ncht = 0; ncht < NCHt; ncht++) {
+        // create numerically stable input, x-max(x)
+        #if NUMERIC_STABLE
+            unpack_reconfig_data_format_srca(cb_in0);
+            ACQ();
+            cb_reserve_back(cb_max, 1);
+            reduce_init_delta<false, PoolType::MAX, ReduceDim::REDUCE_ROW>();
+            for (uint32_t wt = 0; wt < Wt; wt++) {
+                cb_wait_front(cb_in0, wt+1);
+                constexpr uint32_t bcast_scaler0 = 0;
+                reduce_tile<PoolType::MAX, ReduceDim::REDUCE_ROW>(cb_in0, cb_bcast_scaler, wt, bcast_scaler0, dst0);
```

Comment:
could you please explain how given the information provided to `reduce_tile` will allow to ignore values placed in padding? My understanding that once ttnn::max is fixed, "stable softmax" still won't work for large negative values.